### ISSUE TITLE
Merge generates invalid xml when failure message contains "]]&gt;" text

### ISSUE THREAD

**Issue by sarod** [bhovhannes/junit-report-merger#59]
Given a test file TEST-repo.MyTest.xml like this
````xml
<?xml version="1.0" encoding="UTF-8"?>
<testsuite xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://maven.apache.org/surefire/maven-surefire-plugin/xsd/surefire-test-report.xsd" 
name="repro.MyTest" time="2.6" tests="1" errors="0" skipped="0" failures="1">
  <properties/>
  <testcase name="myMethod" classname="repro.MyTest" time="1.573">
    <failure message="failure message with ]]&gt;" type="java.lang.AssertionError">
      java.lang.AssertionError: failure message with ]]&gt;
    </failure>
  </testcase>
</testsuite>
```

junit-report-merger ./combined.xml "TEST-repro.MyTest.xml"
Generates an invalid xml file:
```
<?xml version="1.0"?>
<testsuites failures="1" errors="0" tests="1"><testsuite xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://maven.apache.org/surefire/maven-surefire-plugin/xsd/surefire-test-report.xsd" name="repro.MyTest" time="2.6" tests="1" errors="0" skipped="0" failures="1">
  <properties/>
  <testcase name="myMethod" classname="repro.MyTest" time="1.573">
    <failure message="failure message with ]]>" type="java.lang.AssertionError">
      java.lang.AssertionError: failure message with ]]>
    </failure>
  </testcase>
</testsuite></testsuites>
```

This file is invalid because "]]>" should not appear in xml file outside of xml CDATA. To solve this the > should be escaped as &gt; similarly to what is done in original file

**Comment by bhovhannes** [comment]
Thanks for logging this!
Working on that. I guess this was introduced with the recent update of `xmldom` dependency.

**Comment by bhovhannes** [comment]
This seems similar: https://github.com/xmldom/xmldom/issues/58
Perhaps I'll stop using `xmldom` here and replace it with some sax parser. That will increase performance.

**Comment by bhovhannes** [comment]
Released fix version `2.0.0`.
Release notes: https://github.com/bhovhannes/junit-report-merger/releases/tag/v2.0.0
